### PR TITLE
Retrofit coockiecutter-hypermodern-python 2020.10.15 template

### PR DESCRIPTION
**Pull request by oncleben31**
Update meteofrance-api to use the latest improvement of https://github.com/cjolowicz/cookiecutter-hypermodern-python

Major modifications:

- Add a code of conduct
- Python 3.9 added and used as default for linting
- Remove french translation in contributing instruction.

Contribute to #18

### CODE REVIEW

**Comment by oncleben31** [comment]
CI error with Python 3.7 is linked to [issue #3160 open in poetry repository](https://github.com/python-poetry/poetry/issues/3160#issuecomment-711157947).

File: pyproject.toml [hunk 1]
```diff
@@ -1,54 +1,59 @@
...
 keywords = ["weather", "weather-api", "meteo"]
-readme = "README.rst"
 packages = [
     { include = "meteofrance", from = "src" },
 ]
 classifiers = [
-    "Programming Language :: Python :: 3",
-    "License :: OSI Approved :: MIT License",
-    "Operating System :: OS Independent",
-    "Development Status :: 3 - Alpha",
```
Comment (Quentame) [inline pyproject.toml:18]: I think we should keep this.

Change "Development Status" to `Development Status :: 5 - Production/Stable` perhaps

**Review by Quentame — APPROVED** [review]
Seems good, just waiting the CI to pass before merging.

File: pyproject.toml (hunk 1 above)
Comment (oncleben31) [reply pyproject.toml]: You are right.

**Comment by oncleben31** [comment]
> Seems good, just waiting the CI to pass before merging.

Just have downgrade poetry to 1.0.10 to avoid the CI issue. OK to merge ?

**Pull request merged**